### ISSUE TITLE
HTTPS Support

### ISSUE THREAD

**Issue by frediy** [tricycle/lesswrong#330]
_From [AngryPar...@gmail.com](https://code.google.com/u/104249632091829167509/) on November 09, 2010 13:35:08_

Although FireSheep doesn't steal sessions from LW/reddit accounts by default, it's really easy to add that capability. To mitigate this, LessWrong should be accessible over https. I searched issues for ssl and https but didn't find anything, so I'm creating this issue.

Attachments:
Screenshot of me stealing my own session. 
JavaScript for FireSheep

My main concern would be someone stealing an admin's session at a meetup. Also, I promise not to steal anyone's session besides my own.

**Attachment:** [LessWrong.js Screen shot 2010-11-08 at 6.00.18 PM.png](http://code.google.com/p/lesswrong/issues/detail?id=232)

_Original issue: http://code.google.com/p/lesswrong/issues/detail?id=232_

**Comment by Doom032** [comment]
What happened to this? Did it get stuck because of the problem with certificates? Because maybe Let's Encrypt has made it easier now.

**Comment by cdaloisio** [comment]
I don't believe this was a certificates issue, rather an implementation issue.
This still poses challenges as mentioned by @wezm in this [previous comment](https://github.com/tricycle/lesswrong/issues/330#issuecomment-132416463).
It would also increase server load and slow the response time of the site.

**Comment by binki** [comment]
I really don’t like that the site forces one to login using `http://` action URIs. There’s no protection for users who want to access the site over unprotected networks.

@cdaloisio has that increased load been proven? Keepalive should mitigate the response time for subsequent requests. Most modern and many heavily used websites like Google and reddit use HTTPS without noticeably increasing initial response time.

**Comment by cdaloisio** [comment]
Thanks for the follow-up @binki. It has been a significant amount of time since I commented on this issue, and since then, LessWrong is going to be migrating to www.lesserwrong.com sometime in the future. Please see this post for more details. http://lesswrong.com/lw/pfl/lw_20_open_beta_live/

This codebase is not actively being developed with the intention of retiring it.
